src/app/manager-dashboard/reports/reports.component.ts

```diff
@@ -61,12 +61,12 @@ export class ReportsComponent implements OnInit, OnDestroy {
       switchMap(([ planets, hubs ]) => {
         this.planets = planets;
         this.arrangePlanetData(planets, hubs);
-        return forkJoin([
-          this.activityService.getActivities('resource_activities', 'byPlanet', domain),
-          this.activityService.getActivities('login_activities', 'byPlanet', domain),
-          this.activityService.getAdminActivities({ domain })
-        ]);
-      })
+        return forkJoin(planets.filter((p: any) => p.docType !== 'parentName').map((p: any) =>
+          this.couchService.getUrl('db', { domain: p.localDomain, protocol: 'http', usePort: true })
+          .pipe(catchError(err => of({ [p.code] : false })), switchMap(() => of({ [p.code]: true })))

```

Comment:
This is a problem.  `catchError` will continue along the operator chain, so this means every planet is shown as online because the `switchMap` will still run after the `catchError`.